UnitsNet/GeneratedCode/Quantities/Acceleration.NetFramework.g.cs

```diff
@@ -827,6 +827,62 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
             return string.Format(provider, format, formatArgs);
         }
 
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using <see cref="GlobalConfiguration.DefaultCulture" />.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format)
+        {
+            return ToString(format, GlobalConfiguration.DefaultCulture);
+        }
+
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using the specified format provider, or <see cref="GlobalConfiguration.DefaultCulture" /> if null.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <param name="formatProvider">Format to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format, IFormatProvider formatProvider)
+        {
+            formatProvider = formatProvider ?? GlobalConfiguration.DefaultCulture;
```

Comment:
Can we perhaps extract this code to a helper class? Seems a lot to duplicate N times.

Reply:
👍 done!